### ISSUE TITLE
Ampersand doesn't show up in Macro Name

### ISSUE THREAD

**Issue by minig0d** [notepad-plus-plus/notepad-plus-plus#9230]
### Description of the Issue
Ampersand appears to exist but it is invisible

### Steps to Reproduce the Issue

1. Record a macro of any kind
2. When naming it, include an ampersand (&) in the name Ex: "I like apples & pears"

### Expected Behavior

Ampersand to be visible

### Actual Behavior

It aint (but in shortcuts.xml you can see the ampersand was encoded to &amp; and exists).

### Debug Information

Notepad++ v7.9.1   (64-bit)
Build time : Nov  2 2020 - 01:07:46
Path : C:\Program Files\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS Name : Windows 10 Pro (64-bit) 
OS Version : 2009
OS Build : 19042.630
Current ANSI codepage : 1252
Plugins : BetterMultiSelection.dll CodeAlignmentNpp.dll ComparePlugin.dll DSpellCheck.dll HTMLTag.dll JSMinNPP.dll mimeTools.dll NppConverter.dll NppExport.dll NppSaveAsAdmin.dll NppSnippets.dll NppTaskList.dll RunMe.dll XMLTools.dll _CustomizeToolbar.dll

**Comment by sasumner** [comment]
Same thing happens if you create a `Run` > `Run...` command and save it with such a name.

WORKAROUND:  Double-up on the ampersands, e.g. `I like apples && pears` when saving; will thereafter appear in the menu as `I like apples & pears`

**Comment by JamieSharpe** [comment]
This is by design within Windows (and pretty much all other OS's) for keyboard access.

The ampersand is used to denote the shortcut key for the menu option: [See Microsoft Guidlines.](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/dnacc/guidelines-for-keyboard-user-interface-design#selecting-access-keys)

**Comment by minig0d** [comment]
> This is by design within Windows (and pretty much all other OS's) for keyboard access.
> 
> The ampersand is used to denote the shortcut key for the menu option: [See Microsoft Guidlines.](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/dnacc/guidelines-for-keyboard-user-interface-design#selecting-access-keys)

I just read the guidelines and it doesn't seem to have anything to do with the content of the text utilized in a menu.  It says that the application author should avoid using special characters AS the access key for that entry.  I.e. they do not want you to set your access keys as follows:

**_H_**am Sammich
Peanut Butter **_&_** Jelly Sammich
**_T_**urkey Sammich

Where does it say you should not include an ampersand anywhere within the label?  

Of course, MS has been ditching menus in favor of ribbons, but I just skimmed some of the MS Ribbon's and found several commands containing ampersands (Ex. "Spelling & Grammar" is present in just about the entire office suite.)

**Comment by sasumner** [comment]
I'm going to close this as a feature, not a bug.

As a feature, this allows you to create a menu accelerator.

Example:

I created a macro and I named it `t&est`.
When I drop the macro menu using Alt+m, I see an underlined `e` which means, after dropping the menu that all I have to do is press `e` to run that macro.

![image](https://user-images.githubusercontent.com/30118311/124591764-aac59400-de2a-11eb-8ada-bd9311ba9934.png)

If you truly want an ampersand in the name of your macro, just double it up:  `t&&est` results in:

![image](https://user-images.githubusercontent.com/30118311/124591958-e9f3e500-de2a-11eb-8ef3-9305067c8b56.png)

![image](https://user-images.githubusercontent.com/30118311/124591987-f4ae7a00-de2a-11eb-9775-05e619a46109.png)
